wowup-electron/src/app/pages/my-addons/my-addons.component.html

```diff
@@ -9,7 +9,7 @@
         <mat-label>{{ "PAGES.MY_ADDONS.CLIENT_TYPE_SELECT_LABEL" | translate }}</mat-label>
         <mat-select class="select pointer" [(value)]="selectedInstallationId" (selectionChange)="onClientChange()"
           [disabled]="enableControls === false" [disableOptionCentering]="true">
-          <mat-option [value]="installation.id" *ngFor="let installation of wowInstallations$ | async">
+          <mat-option [value]="installation.id" *ngFor="let installation of trackingEnabled(wowInstallations$) | async">
```

Comment:
Probably dont want to do this, probably want to edit the  ```WarcraftInstallationService``` to provide a new observable based on the current ```wowInstallations$``` filtered to only enabled ones, something like ```enabledWowInstallation$```